src/modules/CRUD/components/Details/index.js

```diff
@@ -11,18 +11,23 @@ export default withRouter(
     settings,
     map,
     fetchFeature,
-    feature,
+    featuresList,
     errors,
   }, {
     match: { params: { layer, id } },
-  }) => ({
-    map,
-    fetchFeature,
-    feature,
-    layer: getLayer(settings, layer),
-    hasError: errors[id],
-    errorCode: errors.code,
-  }))(
+  }) => {
+    const { error = {} } = errors.feature.find(({ featureId }) => featureId === id) || {};
```

Comment:
```js
    const {
      // `error` can be destructured twice : first for having it entirely, then for its properties
      error = {},
      error: { message = '' } = {},
    } = errors.feature.find(({ featureId }) => featureId === id) || {};

    // Using an intermediate const for `feature` allow a more readable return{};
    const feature = featuresList.find(({ identifier }) => (identifier === id)) || {};
    
    return {
      map,
      fetchFeature,
      feature,
      layer: getLayer(settings, layer),

      // As you have defined a default value for undefined `error`, could it really be falsy?
      hasError: !!error,
      errorCode: message,
    };
```

Reply:
In addition I changed `errorCode` by `errorMessage` that makes more sense.